float64 timestamp
uint64 framenumber
float64 omega_x
float64 omega_y
float64 omega_z
float64 head_rate
float64 forw_rate
float64 side_rate

